vllm_gaudi/extension/unified_batch.py

```diff
@@ -0,0 +1,329 @@
+import torch
+import numpy as np
+import habana_frameworks.torch as htorch
+from dataclasses import dataclass
+from vllm_gaudi.extension.unified import HPUUnifiedAttentionMetadata
+import math
+from typing import Optional, Callable, Union
+from vllm_gaudi.extension.logger import logger as init_logger
+
+logger = init_logger()
+
+
+@dataclass
+class UnifiedBatch:
+    req_ids_cpu: list[str]
+    token_ids: torch.tensor
```

Comment:
```suggestion
    token_ids: torch.Tensor
```

I think you should update `torch.tensor` -> `torch.Tensor` in the whole commit
`torch.tensor` is the factory function, not the type and the type annotation should be `torch.Tensor` (capital T).